docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -2286,6 +2295,8 @@ must restart the server for that change to take effect.
 |`fhirServer/persistence/common/updateCreateEnabled`|N|N|
 |`fhirServer/persistence/datasources`|Y|N|
 |`fhirServer/persistence/datasources/<datasourceId>/type`|Y|N|
+|`fhirServer/persistence/datasources/<datasourceId>/jndiName`|Y|Y|
+|`fhirServer/persistence/datasources/<datasourceId>/currentSchema`|Y|Y|
```

Comment:
are these really dynamic?  if so, are some of the other datasource properties dynamic now as well, or just these ones?

Reply:
I guess jndiName is dynamic because if you change this then it will use a fresh connection pool (not just grab from the existing one)

Reply:
I think it's just these, but underneath the datasource caching isn't dynamic